content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,167 @@
+---
+reviewers:
+- michmike
+- patricklang
+title: Adding Windows nodes
+min-kubernetes-server-version: 1.17
+content_template: templates/tutorial
+weight: 30
+---
+
+{{% capture overview %}}
+
+You can use Kubernetes to run a mixture of Linux and Windows nodes, so you can mix Pods that run on Linux on with Pods that run on Windows. This page shows how to register Windows nodes to your cluster.
```

Comment:
add the feature gate of beta here please
{{< feature-state for_k8s_version="v1.18" state="beta" >}}